GithubDemo/Maths.cs

```diff
@@ -15,5 +15,9 @@ public class Maths {
         public int Multiply(int a, int b) {
             return a * b;
         }
+
+        public int Divide(int a, int b) {
+            return a / b;
```

Comment:
Please check the second parameter for zero to avoid runtime error.

Reply:
Taken care.